### DISCUSSION TITLE
De un día para otro el emulador dejó de funcionar

### DISCUSSION THREAD

**Discussion by Poisonworms** [mupen64plus-ae/mupen64plus-ae#1058] [Q&A]
No entiendo que pasó, llevo mucho tiempo usando el emulador sin problemas, pensé que era la versión del Android, pero no, en otro dispositivo con otra versión pasa lo mismo, simplemente no me deja usar la opción de selección de carpeta, y no puedo añadir ni una sola ROM, por lo tanto queda inutilizable el emulador, la verdad no entiendo que sucedió ni como, simplemente que no puedo hacer nada
![IMG_20220819_012123](https://user-images.githubusercontent.com/111558627/185555992-d2230b52-dac2-44ec-b99a-a31f4ff45338.jpg)

**Comment by fzurita** [comment]
Esa foto parece que es de una versión muy antigua. Puedes probar con la versión más nueva?

**Reply by Poisonworms** [reply]
De hecho precisamente tengo el mismo problema con la última versión del emulador en los 2 dispositivos, la versión 3.0.322 no la ocupaba porque siempre me funcionaba la 3.0.280, pero también me pasa lo mismo, simplemente no me deja seleccionar carpeta, y cuando le doy en seleccionar archivo nunca lo hace, se reinicia el emulador en cuanto intento cargar el archivo

**Comment by fzurita** [comment]
Que celular tienes? Normalmente los usuarios no tienen este problema.

**Reply by Poisonworms** [reply]
Un dispositivo donde me pasa es en la nintendo switch con Android, ya que es bastante cómodo jugar con los joy cons, y el otro es un Xiaomi Redmi 9T, yo también no entiendo porque pasa eso, pero al final ya me dejó así

**Reply by Poisonworms** [reply]
Acabo de checar, creo que el problema si es la versión de Android del celular con el MIUI que tiene el Xiaomi, acabo de checar en la Switch y ahí no sale ese error, voy a resetear el celular y probar si vuelve a funcionar el emulador
